src/ServiceControl.AcceptanceTesting/InfrastructureConfig/ConfigureEndpointLearningTransport.cs

```diff
@@ -46,6 +45,7 @@ public Task Cleanup()
         }
 
         public string Name => "Learning";
```

Comment:
since these names are also used in tests, is there any reason to not make them static constants or an enum in order to prevent typos?

Reply:
Mostly just that I didn't see a lot of benefit in doing that vs just making the changes. If there was a typo, it would be pretty immediately obvious since it would no longer match the value in a manifest file, and the the persister or transport would not be found, and the tests would fail in a rather dramatic fashion.